apps/frontend/src/app/containers/userProfile/UserEditProfile.tsx

```diff
@@ -23,15 +30,24 @@ export const UserEditProfile = ({ originalUser, toggleEdit }: IProps): ReactElem
 
     const handleSubmit = async (event: MouseEvent<HTMLButtonElement>) => {
         event.preventDefault();
-        setPatching(true);
+        if (validateRegisterUser(user).isValid) {
+            setPatching(true);
+            const { response, isError } = await updateUser({ ...user, latestUpdate: Date.now().toString() });
+            if (response) {
+                toggleEdit();
+            } else if (isError?.response?.status === 406) {
+                setErrorMessages({
+                    nameErrorMsg: errorMessages.nameErrorMsg,
+                    emailErrorMsg: 'En bruker med denne eposten finnes allerede',
+                    phoneErrorMsg: errorMessages.phoneErrorMsg
+                });
```

Comment:
Does the "unique" error not get picked up the same way as "empty field"? Confused as to why it's handled differently :P

Reply:
The unique error stems from a user with the same email being in the database, while the empty whield stems from the user not entering an input

Reply:
Aight 👍